src/Faker/Provider/Color.php

```diff
@@ -113,4 +113,26 @@ public static function colorName()
     {
         return static::randomElement(static::$allColorNames);
     }
+
+    /**
+     * @example '340,50,20'
```

Comment:
What do you think about adding a `@return` annotation with `array` as type here?

Reply:
Its more like `string`. But I added it :)